packages/agents/sequencer/src/bindings/server/index.ts

```diff
@@ -111,14 +112,29 @@ export const bindServer = async (): Promise<FastifyInstance> => {
       try {
         const bid = request.body;
         requestContext.transferId = bid.transferId;
-        await storeFastPathData(bid, requestContext);
+        // Validate Input schema
+        const validateInput = ajv.compile(BidSchema);
+        const validInput = validateInput(bid);
+        if (!validInput) {
+          const msg = validateInput.errors?.map((err: any) => `${err.instancePath} - ${err.message}`).join(",");
+          throw new DataInvalid({
```

Comment:
Schema validation errors should not throw 500, I think using the validation strategy above will take care of that.